### DISCUSSION TITLE
Digest Redux

### DISCUSSION THREAD

**Discussion by jfarseneau** [jfarseneau/antennas#89] [Announcements]
Hopefully this time I've fixed things rather than breaking them further?

The main change in this one is restoring digest auth. It had been broken when I changed from using an older HTTP client library to Axios, which does not support digest auth. There's a little wrapper that has been added in now to support that authentication format.

I've also taken [davmc123](https://github.com/davmc123)'s suggestion and added `--yes` to the Docker `npx` command so folks don't get prompted on Docker run.

## What's Changed
* Add "--yes" to run command by @jfarseneau in https://github.com/jfarseneau/antennas/pull/87
* Restore digest auth by @jfarseneau in https://github.com/jfarseneau/antennas/pull/88

**Full Changelog**: https://github.com/jfarseneau/antennas/compare/v4.1.2...v4.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/jfarseneau/antennas/releases/tag/v4.2.0'>Digest Redux</a>.</em>

**Comment by Jimbob-B** [comment]
I don't really want to ask this but has digest auth broken again? My Antennas now refuses to connect to my TVHeadEnd on the same Raspberry Pi running bullseye 64 bit. Things I've tried include changing the user and password, removing special characters from the password, downgrading node, completely fresh antennas install with just my config changes. Scratching my head a bit for what has happened ... noticed issues when I went to node 20 and then reverted back to 19 and that didn't fix it.

Also tried the WIP v5.0 but couldn't seem to get that to work, so deleted the install.

One thing to note, I build tvheadend from their git ... could that be causing the issue? Currently on version 4.3-2132~gdbf973307.

Thanks!

**Reply by jfarseneau** [reply]
Ugh, that's very possible. Since I haven't released a new update, I'd imagine it's more on the TVHeadEnd side.

I'll give the new version of TVHeadEnd a whirl when I get a bit of free time for it. Thanks for raising this!
